Source/inv.cpp

```diff
@@ -1781,10 +1781,8 @@ void AutoGetItem(int pnum, int ii)
 		dropGoldValue = 0;
 	}
 
-	if (ii != MAXITEMS) {

```

Comment:
why did you remove this? :P

Reply:
```
if (pnum == myplr && pcurs >= CURSOR_FIRSTITEM) {
	item[MAXITEMS] = plr[myplr].HoldItem;
	AutoGetItem(myplr, MAXITEMS);
}
```
`if (ii != MAXITEMS)` was part of the hack added in 1.02 to prevent items disappearing from cursor when entering a portal. They store the item into the temporary slot and added that check to `AutoGetItem` so it will go into inventory. This check is pointless once the above code is removed because items on the ground can only be in slots 0-126, slot 127 is otherwise impossible and only used in the pack player code.